mods/bones/init.lua

```diff
@@ -87,6 +108,10 @@ local bones_def = {
 			return
 		end
 
+		if not player:is_player() then
+			dropContents(pos)
```

Comment:
Shouldn't this be followed by a `return`?

Reply:
return is added now.